pkg/hintrunner/zero/zerohint_ec.go

```diff
@@ -566,8 +566,8 @@ func createReduceEd25519Hinter(resolver hintReferenceResolver) (hinter.Hinter, e
 // It also assigns `slope`, `x`, `y` and `new_x` in the current scope
 // so that they are available in the current scope for EcDoubleAssignNewYV1 hint
 //
-// This implementation is valid for both EcDoubleAssignNewX V1 and V4, only the operander differs
-// with `point` used for V1 and `pt` used for V4
+// This implementation is valid for EcDoubleAssignNewX V1,V2 and V4, only the operander differs
+// with `point` used for V1,V2 and `pt` used for V4
```

Comment:
Can you add that with V2 , SECP is supposed to be already in scope (contrary to V1 which adds it in scope as well

Reply:
Done! ✅